xmtp_mls/src/storage/encrypted_store/group_message.rs

```diff
@@ -77,6 +86,61 @@ where
     }
 }
 
+#[repr(i32)]
+#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq, Eq, FromSqlRow, AsExpression)]
+#[diesel(sql_type = diesel::sql_types::Integer)]
+pub enum ContentType {
+    Unknown = 0,
+    Text = 1,
+    GroupMembershipChange = 2,
+    GroupUpdated = 3,
+}
+
+impl ContentType {
+    pub fn from_string(type_id: &str) -> Self {
+        match type_id {
+            text::TextCodec::TYPE_ID => Self::Text,
+            membership_change::GroupMembershipChangeCodec::TYPE_ID => Self::GroupMembershipChange,
+            group_updated::GroupUpdatedCodec::TYPE_ID => Self::GroupUpdated,
+            _ => Self::Unknown,
+        }
+    }
+
+    pub fn to_string(&self) -> &'static str {
```

Comment:
It would be more rustlike to `impl std::fmt::Display` here, which provides a `to_string`. 

implementing display allows it to be easily `printed` and logged

Reply:
🙏  added https://github.com/xmtp/libxmtp/pull/1435/commits/dc6428cd4633aab9f49ff2408921eed1461b714d